code/game/objects/items/cards_ids.dm

```diff
@@ -460,21 +464,6 @@
 			powergaming.update_label()
 			powergaming.update_appearance()
 
-/*
-Usage:
-update_label()
-	Sets the id name to whatever registered_name and assignment is
-
-update_label("John Doe", "Clowny")
-	Properly formats the name and occupation and sets the id name to the arguments
-*/
-/obj/item/card/id/proc/update_label(newname, newjob)
-	if(newname || newjob)
-		name = "[(!newname)	? "identification card"	: "[newname]'s ID Card"][(!newjob) ? "" : " ([newjob])"]"
-		return
-
-	name = "[(!registered_name)	? "identification card"	: "[registered_name]'s ID Card"][(!assignment) ? "" : " ([assignment])"]"
-
```

Comment:
What does the removal of this do? Where is the name of the item updated with this removed?

Reply:
I've looked into it and this would require some pathing changes. I'm going to stick with the 'Indentification card" for the first TM and see if its too obscure